wagtail/users/views/groups.py

```diff
@@ -76,6 +76,9 @@ class CreateView(PermissionPanelFormsMixin, generic.CreateView):
     page_title = _("Add group")
     success_message = _("Group '%(object)s' created.")
 
+    def get_page_subtitle(self):
+        return ""
```

Comment:
I'm retaining the behaviour of showing "Add group" as the title (without subtitle) instead of the generic view's "Add" title + `capfirst({model_name})` subtitle, though we probably should change this for consistency with the generic view.

One downside I see with this is that in other languages, the correct form might be the other way around (`model_name` + "Add").

However, the assumption of action + `model_name` has been the case with the generic views so I suppose it's alright... I think ideally we should not have a design where `page_title` and `page_subtitle` is shown side-by-side to form a phrase, e.g. "**Editing** Muddy Waters", as this is the root cause of the aforementioned issue with other languages

Reply:
I think the capfirst stuff has caused us translation issues in the past, probably best to avoid unless there is a critical reason to add it.